README.md

```diff
@@ -138,6 +139,12 @@ For splitting (segmenting) applications configuration files are used. For each p
 
 There are more practical [examples](examples/README.md) in the repository. Practical guides are provided in the [publications](#publications) section.
 
+## Integrations
+
+Jitar provides integration examples for the following frameworks:
+- [react-jitar](examples/4-integrations/1-react/README.md)
+- [vue-jitar](examples/4-integrations/2-vue/README.md)
+
```

Comment:
We are working on integrations with more frontend and meta frameworks, so this list will grow over time.

Reply:
Processed